### ISSUE TITLE
error: failed to launch REPL process: process launch failed: unable to locate debugserver

### ISSUE THREAD

**Issue by AxiosCros** [tensorflow/swift#9]
``` shell
$ export PATH=/Library/Developer/Toolchains/swift-latest/usr/bin:"${PATH}"

$ swift
error: failed to launch REPL process: process launch failed: unable to locate debugserver

$ cd /Library/Developer/Toolchains/swift-latest/usr/bin
$ ls
lldb			swift-autolink-extract	swift-demangle		swift-run		swiftc
lldb-mi			swift-build		swift-format		swift-stdlib-tool
swift			swift-build-tool	swift-package		swift-test

$ ./swift
error: failed to launch REPL process: process launch failed: unable to locate debugserver

```

show the "verify code signature"
```
“Swift for TensorFlow Development Snapshot 2018-04-26” does not have a valid signature: code failed to satisfy specified code requirement(s)
```

**Comment by dan-zheng** [comment]
I'm assuming you're running macOS. What version of macOS are you running?
Have you installed Xcode 9.3?

**Comment by AxiosCros** [comment]
![image](https://user-images.githubusercontent.com/13862149/39391147-90b01206-4ad1-11e8-829b-9939175d4f10.png)

**Comment by dan-zheng** [comment]
Sorry, I wasn't able to replicate your error. I tested on a nearly fresh installation of Mac:
* Install Xcode 9.3 and run `sudo xcode-select -s /Applications/Xcode.app`, then `sudo xcode-select --install`.
* Install the Xcode toolchain from [here](https://github.com/tensorflow/swift/blob/master/Installation.md#pre-built-packages).
* Run `export PATH=/Library/Developer/Toolchains/swift-latest/usr/bin:"${PATH}"`.
* Run `swift` to launch the REPL.

If you provide more information about your environment, I can try to help.

**Comment by lattner** [comment]
What happens when you just run "/usr/bin/lldb"?

**Comment by yangcheng** [comment]
I have the same error when I have multiple xcode installations. Remove and recreate xcode link to 9.3 fixed it

**Comment by AxiosCros** [comment]
macOS 10.13.4 ;  XCode9.3 ; 

``` shell
$ sudo xcode-select -s /Applications/Xcode.app
$ sudo xcode-select --install
xcode-select: error: command line tools are already installed, use "Software Update" to install updates
$ swift
Welcome to Apple Swift version 4.1 (swiftlang-902.0.48 clang-902.0.39.1). Type :help for assistance.

$ /Library/Developer/Toolchains/swift-latest/usr/bin/swift
Welcome to Swift version 4.2-dev (LLVM 04bdb56f3d, Clang b44dbbdf44). Type :help for assistance.
```
@dan-zheng  It‘s already solved. thank you.


```shell
$ /usr/bin/lldb
(lldb)  <wait for input>
```
@lattner

**Comment by 4ei-davidcittadini** [comment]
How was this resolved?  I just installed Xcode 10.1 on macOS Mojave and Swift for TensorFlow and I get the error "error: failed to launch REPL process: process launch failed: unable to locate debugserver" when I try to 'swift'.

**Comment by 4ei-davidcittadini** [comment]
I resolved my problem.  If I ran the standard swift REPL it was fine, if I ran the Swift for TensorFlow I got the above error.  However, once I checked Preferences->Locations->Command Line Tools, and made sure that the Xcode 10.1 tools where selected it worked find.  For some reason it was unselected.
